flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -326,4 +327,51 @@ public void testTwoSinksInDisjointedDAG() throws Exception {
     Assert.assertEquals("rightTable", rightTable.currentSnapshot().summary().get("direction"));
   }
 
+  @Test
+  public void testOverrideWriteConfigWithUnknownDistributionMode() {
+    AssertHelpers.assertThrows("Override write configuration.",
```

Comment:
This context should explain what you're testing a bit better, like "Should fail with invalid distribution mode"

Reply:
Learned, thank you.